README.md

```diff
@@ -82,6 +80,14 @@ npm run test-event
 
 ## Deployment
 
```

Comment:
Sorry - I know it's a pain - but Can we add a line about how this is hooked up to CI/CD so merging to a branch automatically deploys?

Reply:
OK I have updated README.md